phpstan.neon.dist

```diff
@@ -1,3 +1,5 @@
 parameters:
     ignoreErrors:
         - '#Contao\\Image\\DeferredImage::__construct\(\) does not call parent constructor#'
+        - '#unknown class Gmagick#'
+        - '#class Gmagick\S* not found#'
```

Comment:
@leofeyer @bytehead Is this the correct way to “fix” the phpstan errors here?

```
 ------ -------------------------------------------------------------------
  Line   src/Resizer.php
 ------ -------------------------------------------------------------------
  129    Caught class GmagickException not found.
  130    Call to method getMessage() on an unknown class GmagickException.
  131    Throwing object of an unknown class GmagickException.
  135    Instantiated class Gmagick not found.
  136    Call to method getcolor() on an unknown class GmagickPixel.
  136    Call to method newimage() on an unknown class Gmagick.
  136    Instantiated class GmagickPixel not found.
  137    Call to method setimageformat() on an unknown class Gmagick.
  138    Call to method getImageBlob() on an unknown class Gmagick.
 ------ -------------------------------------------------------------------
```

Reply:
Basically yes. But why does PHPStan not find the class?

Reply:
`Gmagick` is not installed in the CI environment, and AFAIK it should not be installed together with `Imagick` at the same time. Is there a way to tell phpstan how the Gmagick classes would look like? Stub files?

Reply:
I guess @bytehead can answer this question better than me.

Reply:
There is actually a way to provide stub files for `Imagick`:
```
parameters:
	stubFiles:
		- stubs/Imagick.stub
```

Reply:
See https://phpstan.org/user-guide/stub-files as well 🙃

Reply:
Doesn’t seem to work 😢 https://github.com/contao/image/pull/70/checks#step:6:38

Reply:
Maybe because of `You're running a really old version of PHPStan.️`?

Reply:
Fixed. With the new version the stub file is not even needed :)